src/LiveComponent/assets/src/set_deep_data.ts

```diff
@@ -64,9 +75,11 @@ export function doesDeepPropertyExist(data, propertyPath) {
  */
 export function normalizeModelName(model) {
     return model
+        .replace(/\[]$/, '')
```

Comment:
Can you add a test to the bottom of `set_deep_data.test.ts` to test & explain what this does?

Reply:
I will paste here comment as I'm adding it:

> Names ending in "[]" represent arrays in HTML.
> To get normalized name we need to ignore this part.
> For example: "user[mailing][]" becomes "user.mailing" (and has array typed value)

I have also added simple test case